libs/blocks/editorial-card/editorial-card.css

```diff
@@ -248,3 +263,24 @@
 
 .editorial-card.footer-align-left .action-area { justify-content: start; }
 .editorial-card.footer-align-center .action-area { justify-content: center; }
+
+
+/* Tablet up */
+@media screen and (min-width: 600px) {
+  .editorial-card .vp-media .mobile-only,
+  .editorial-card .vp-media .desktop-only {
```

Comment:
I like the consistency, but I think this line is not really needed, `desktop-only` is already hidden on mobile.
<img width="440" alt="Screenshot 2024-09-04 at 21 03 34" src="https://github.com/user-attachments/assets/06fc9100-1d93-4566-be17-1eb10973be3e">.

We can keep the code for consistency purposes however.